spyder/plugins/application/container.py

```diff
@@ -284,73 +279,105 @@ def _check_updates_ready(self):
 
```

Comment:
```suggestion
        box.setTextFormat(Qt.RichText)

```

This is necessary to make the new link to our docs clickable. But please test that that's really the case @mrclary.